src/WCSimStackingAction.cc

```diff
@@ -26,32 +26,40 @@ G4ClassificationOfNewTrack WCSimStackingAction::ClassifyNewTrack
   G4String WCIDCollectionName = DetConstruct->GetIDCollectionName();
   G4ClassificationOfNewTrack classification    = fWaiting;
   G4ParticleDefinition*      particleType      = aTrack->GetDefinition();
-  
+
 
   // Make sure it is an optical photon
   if( particleType == G4OpticalPhoton::OpticalPhotonDefinition() )
     {
       G4double photonWavelength = (2.0*M_PI*197.3)/(aTrack->GetTotalEnergy()/eV);
-      G4double ratio = 1./(1.0-0.25);
+      // MF : translated from skdetsim : better to increase the number of photons
+      // than to throw in a global factor  at Digitization time !
+      //G4double ratio = 1./(1.0-0.25);
+      G4double ratio = 0.5;
+      // XQ: get the maximum QE and multiply it by the ratio
+      // only work for the range between 240 nm and 660 nm for now 
+      // Even with WLS
       G4double wavelengthQE = 0;
       if(aTrack->GetCreatorProcess()==NULL) {
-	wavelengthQE  = DetConstruct->GetPMTQE(WCIDCollectionName,photonWavelength,1,240,660,ratio);
+	if(!DetConstruct->GetIsCombinedPMTCollectionDefined())
+	  wavelengthQE  = DetConstruct->GetPMTQE(WCIDCollectionName,photonWavelength,1,200,700,ratio);
```

Comment:
```suggestion
	  wavelengthQE  = DetConstruct->GetPMTQE(WCIDCollectionName,photonWavelength,1,240,660,ratio);
```
Another change that could be in the Travis CI. And seemingly, this one doesn't affect the OD. So please revert